server/pulp/server/managers/consumer/group/cud.py

```diff
@@ -78,9 +66,7 @@ def create_consumer_group(group_id, display_name=None, description=None, consume
         try:
             collection.insert(consumer_group, safe=True)
         except DuplicateKeyError:
-            raise pulp_exceptions.PulpCodedValidationException(
-                [PulpCodedException(error_codes.PLP1004, type=ConsumerGroup.collection_name,
-                                    object_id=group_id)])
+            raise pulp_exceptions.DuplicateResource(group_id), None, sys.exc_info()[2]
```

Comment:
the problem itself was in the `except` line, it would look like this http://fpaste.org/192664/42538499/ but i decided to go with the 2nd option since it is a pattern in pulp